code/game/objects/items/crayons.dm

```diff
@@ -665,7 +718,7 @@
 
 		return
 
-	if(isobj(target))
+	if(isobj(target) && !istype(target, /obj/effect/decal/cleanable/crayon/gang))
```

Comment:
return of the artificially-blackened punji stick and/or disposal chute hallway kill traps: gang edition

Reply:
gang tags are forced to render over everything

Reply:
that's exactly the point

you can spam them to make an area appear artificially dark, concealing dangerous things like punji sticks and disposal chutes that will fire you directly into the SM

watch some of imsxz's maint trap murderboning videos; they're SCARILY effective

Reply:
either you're gonna have a really bad time cleaning the gang tag, or you get vietconged and thrown out of the airlock